torch_frame/utils/split.py

```diff
@@ -5,25 +5,35 @@
 
 
 def generate_random_split(length: int, seed: int, train_ratio: float = 0.8,
-                          val_ratio: float = 0.1) -> np.ndarray:
+                          val_ratio: float = 0.1,
+                          include_test: bool = True) -> np.ndarray:
     r"""Generate a list of random split assignments of the specified length.
     The elements are either :obj:`0`, :obj:`1`, or :obj:`2`, representing
     train, val, test, respectively. Note that this function relies on the fact
     that numpy's shuffle is consistent across versions, which has been
     historically the case.
     """
-    assert train_ratio + val_ratio < 1
+    if include_test:
+        assert train_ratio + val_ratio < 1
```

Comment:
If we're not including test, then train and val should add up to 1 (up to an eps), right? Or are we ignoring val_ratio completely ? in which case you can just check that `train_ratio < 1`.